M2/Macaulay2/packages/VirtualResolutions/tests.m2

```diff
@@ -200,6 +211,7 @@ TEST ///
     assert(multigradedRegularity(S, I) == {{1,5},{2,2},{4,1}})
 ///
 
+-- This test currently fails due to a bug in tateResolutions
 ///
```

Comment:
Ahh, this is fixed in TateOnProducts now. @DanGrayson could you remove the comment and enable that test in a commit to development? I'd rather not make a new PR for one line.

Reply:
okay  :  commit 6cbb03398cd607181c78ab1c69663a7e94abac64